js/src/Ice/PropertyNames.js

```diff
@@ -1,35 +1,18 @@
 // Copyright (c) ZeroC, Inc. All rights reserved.
 
-// Generated by makeprops.py from PropertyNames.xml, Tue Jun  4 11:30:47 2024
+// Generated by makeprops.py from PropertyNames.xml, Wed Jun 19 10:57:46 2024
 
 // IMPORTANT: Do not edit this file -- any edits made here will be lost!
 
 /* eslint comma-dangle: "off" */
 /* eslint array-bracket-newline: "off" */
 /* eslint no-useless-escape: "off" */
 
-import { Property } from "./Property.js";
+import { Property } from "./Property";
 export const PropertyNames = {};
-PropertyNames.IceProps = [
+PropertyNames.IceProps =
```

Comment:
You can run `npx prettier --write src/Ice/PropertyNames.js` in js to reformat this file. I didn't have time to add the style checker to CI yet.